src/components/organisms/cards/cards.html

```diff
@@ -0,0 +1,45 @@
+<div class="organisms-cards">
+  <div class="container">
+    <div class="cards">
+      <div class="card">
+        <p class="cardTitle">CONCEPT</p>
+        <div class="triangle"></div>
+        <figure class="card-img">
+          <img src="../../../img/card-1.jpg" class="card-img-top" alt="...">
```

Comment:
```suggestion
          <img src="../../../img/card-1.jpg" class="card-img-top" alt="">
```
画像に意味がないときは `alt` は空文字列でいきましょう

Reply:
対応しました

7447352298afd2ee693fdef833d126ae0ec4f6c9